packages/instantsearch.js/src/widgets/frequently-bought-together/frequently-bought-together.tsx

```diff
@@ -0,0 +1,220 @@
+/** @jsx h */
+
+import { createFrequentlyBoughtTogetherComponent } from 'instantsearch-ui-components';
+import { Fragment, h, render } from 'preact';
+
+import TemplateComponent from '../../components/Template/Template';
+import connectFrequentlyBoughtTogether from '../../connectors/frequently-bought-together/connectFrequentlyBoughtTogether';
+import { prepareTemplateProps } from '../../lib/templating';
+import {
+  getContainerNode,
+  createDocumentationMessageGenerator,
+} from '../../lib/utils';
+
+import defaultTemplates from './defaultTemplates';
+
+import type {
+  FrequentlyBoughtTogetherWidgetDescription,
+  FrequentlyBoughtTogetherConnectorParams,
+  FrequentlyBoughtTogetherRenderState,
+} from '../../connectors/frequently-bought-together/connectFrequentlyBoughtTogether';
+import type { PreparedTemplateProps } from '../../lib/templating';
+import type { Template, WidgetFactory, Hit, Renderer } from '../../types';
+import type { RecommendResultItem } from 'algoliasearch-helper';
+import type {
+  RecommendClassNames,
+  FrequentlyBoughtTogetherProps as FrequentlyBoughtTogetherUiProps,
+} from 'instantsearch-ui-components';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'frequentlyBoughtTogether',
```

Comment:
This will require to update one test.

```suggestion
  name: 'frequently-bought-together',
```